modules/member_iam/outputs.tf

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "roles" {
```

Comment:
Outputs which simply pass back inputs are not useful and considered an anti-pattern. At a minimum these outputs should be constructed from the resource outputs.

Or simply return the entire resource as an output (`value = google_project_iam_member.project_iam_member`).